control-plane/plugin/src/resources/pool.rs

```diff
@@ -60,18 +61,77 @@ impl GetHeaderRow for openapi::models::Pool {
     }
 }
 
+#[derive(Debug, Clone, clap::Args)]
+/// Arguments used when getting nodes.
+pub struct GetPoolsArgs {
+    /// Id of the node
+    #[clap(long)]
+    node_id: Option<NodeId>,
+
+    /// Filter pools on labels
+    #[clap(long)]
+    labels: Option<String>,
```

Comment:
To match the same term kubectl uses?

```suggestion
    selector: Option<String>,
```